src/Styleguide/Components/Tabs.tsx

```diff
@@ -76,7 +83,10 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
 interface TabProps {
   name: string
 }
-export class Tab extends React.Component<TabProps> {
+interface PreloadLinkTabProps extends TabProps, PreloadLinkProps {
+  children: never
```

Comment:
This doesn't work. @alloy have any advice here? It seems that typescript isn't using this to check the type of the children.